src/ws/update-motd

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+
+# syntax: activate-motd [port [hostname [ipaddr [protocol]]]]
```

Comment:
Should be "update-motd"?

Reply:
lol renamed the file, thanks for catching